sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTestCommon/Microsoft.Azure.Batch.IntegrationTestCommon.csproj

```diff
@@ -15,7 +15,7 @@
     <PackageReference Include="System.IO.Compression" />
     <PackageReference Include="WindowsAzure.Storage" />
     <PackageReference Include="Microsoft.Azure.Management.Batch" />
-    <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" VersionOverride="[3.14.0, 4.0.0)" />
+    <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" VersionOverride="[3.14.2, 4.0.0)" />
```

Comment:
Given these are test projects I would honestly not special case them and would just use 4.5.1 for them assuming the tests work with that version.

Reply:
@matthchr could you take a look at the changes under batch?

Reply:
They do not, unfortunately.   The management plane causes build issues due to a dependency on 3.14.2

Reply:
OK.

Reply:
Management plane in this case means "these specific tests consume the management libraries for their services in order to dynamically control resources for tests"  not  "something in the management plane caused issues."    Apologies for the sloppy explanation.